grp/classic.gi

```diff
@@ -1525,22 +1531,11 @@ BindGlobal( "OmegaZero", function( d, q )
         n[ d+1-i ][ d-i ]:= o;
       od;
 
-      if q mod 2 = 0 then
-        # $x = x_{\epsilon_1 - \epsilon_m}(1) x_{-\alpha_1}(1)$
-        x1:= IdentityMat( d, f );
-        x1[1][m]:= o;
-        x1[ m+2 ][d]:= o;
-        x2:= IdentityMat( d, f );
-        x2[ m+1 ][m]:= o;
-        x2[ m+2 ][m]:= o;
-        x:= x1 * x2;
```

Comment:
Optional: now x1 and x2 are not used anymore in this function, and could be removed from the `local` list above.